docs/express-or-core.qmd

```diff
@@ -0,0 +1,267 @@
+---
+title: Choosing a syntax
+editor:
+  markdown:
+    wrap: sentence
+---
+
+Now that you are familiar with [the differences between Shiny Express and Shiny Core](express-vs-core.qmd), you might be wondering how to choose between them.
+
+In this article, we'll suggest some guidelines, but it's important to note that there are not many hard and fast rules.
+There is a lot of overlap between the capabilities of the two syntaxes, so feel free to choose whichever feels more natural and comfortable for you.
+
+Shiny Express is designed to get you up and running as quickly as possible.
+It's also designed to let you author your app with a minimum of boilerplate.
+As a result, it really shines in the early stages of both the learning journey and the lifecycle of an app.
+
+Compared to Express, Core enforces a more structured approach that requires more initial effort to use.
+However, that investment has its own payoff.
+
+The bulk of the rest of this article will focus on the advantages of Shiny Core.
+This isn't because we think it's better, but because Shiny Express's advantages---being more approachable and more concise---are fairly self-evident, while Shiny Core's advantages are more subtle.
```

Comment:
```suggestion
This isn't because we think it's better overall, but because Shiny Express's advantages---being more approachable and more concise---are fairly self-evident, while Shiny Core's advantages are more subtle.
```